public/index.php

```diff
@@ -13,7 +13,7 @@
 }
 
 if ($trustedProxies = $_SERVER['TRUSTED_PROXIES'] ?? $_ENV['TRUSTED_PROXIES'] ?? false) {
-    Request::setTrustedProxies(explode(',', $trustedProxies), Request::HEADER_X_FORWARDED_ALL ^ Request::HEADER_X_FORWARDED_HOST);
+    Request::setTrustedProxies(explode(',', $trustedProxies), Request::HEADER_X_FORWARDED_AWS_ELB);
 }
 
```

Comment:
I'd handle CF forwarded proto too .. even if we are not using it nowadays I'd make the code ready to do it :

```php
$_SERVER['HTTP_X_FORWARDED_PROTO'] = $_SERVER['HTTP_CLOUDFRONT_FORWARDED_PROTO'] ?: $_SERVER['HTTP_X_FORWARDED_PROTO'];

```

Reply:
done